mlc_llm/core.py

```diff
@@ -137,9 +130,7 @@ def convert_build_args_to_argparser() -> argparse.ArgumentParser:
             # boolean arguments do not need to specify `type`
             args.add_argument(field_name, default=field.default, **kwargs)
         else:
-            args.add_argument(
-                field_name, type=field.type, default=field.default, **kwargs
-            )
+            args.add_argument(field_name, field.type, default=field.default, **kwargs)
```

Comment:
hi junru, @junrushao , maybe some typo in convert_build_args_to_argparser. 

https://github.com/mlc-ai/mlc-llm/blob/9c77d19a4212323d05033cd077d6a9b9b62ddc50/mlc_llm/core.py#L121-L134

should be 

`args.add_argument(field_name, type=field.type, default=field.default, **kwargs)`